teacher/templates/teacher_base.html

```diff
@@ -1,74 +1,168 @@
-{% load static %}
 <!DOCTYPE html>
```

Comment:
There seems to be a fair amount of redundancy between the base templates (navigation, dependencies, some of the javascript, etc.). All of that should get factored out to a single template that the other templates depend on.

Reply:
I agree, we will talk about this and take care of it in the next couple days

Reply:
Only have one base template now 7bb73e8ca4c21297b7237dbf044b08a5f0cf164b